lib/scrabble.rb

```diff
@@ -1,18 +1,35 @@
 class Scrabble
+  attr_reader :word
+
+  def initialize
+    @word = nil
+  end
 
   def score(word)
-    1
+    @word = word.upcase!
+    word = sub_points(@word)
```

Comment:
Naming things is notoriously difficult in programming, but take the time to be descriptive. `sub_points` doesn't really return a word; it returns points, so `points` would probably be a better variable name here.

Seeing `word` on the left hand side of these assignments three times makes me wonder what each of these methods is really doing. Seeing `word`, `points`, and `score` on the other hand tells me that we're taking a word, turning t into points, and then turning those points into a score.